### PR TITLE
LPS-125884

### PR DESCRIPTION
**Pull request by brunoquei2105**

### CODE REVIEW

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -2,6 +2,7 @@ check-token=Check Image Token
 maximum-height=Maximum Height
 maximum-width=Maximum Width
 organizations-types-help=Set this JSON object to change the organizations types. Add a new property to the JSON object to create a new organization type.
+unable-to-create-organization-because-the-maximum-number-of-organizations-has-been-reached=qqaximum number of organizations has been reached
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: @brunoquei2105 looks like a typo in this translation.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties (hunk 1 above)
Comment (brunoquei2105) [reply modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: Yes, i'll fix.

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

File: modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -2,6 +2,7 @@ check-token=Check Image Token
 maximum-height=Maximum Height
 maximum-width=Maximum Width
 organizations-types-help=Set this JSON object to change the organizations types. Add a new property to the JSON object to create a new organization type.
+unable-to-create-organization-because-the-maximum-number-of-organizations-has-been-reached=unable to create organization because the maximum number of organizations has been reached.
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-api/src/main/resources/content/Language.properties:5]: @brunoquei2105 need to capitalize `unable` in the translation.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**